---
title: Bloc 'Catch' jamais atteint, car '<name1>'hérite de'<name2>'
ms.date: 07/20/2015
f1_keywords:
- BC42029
- vbc42029
helpviewer_keywords:
- BC42029
ms.assetid: 4bc84a84-ec7e-49db-8794-4fb71d36d5c0
ms.openlocfilehash: 59dfa9cbc80bc9cc3faf07923d0d579070750426
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: HT
ms.contentlocale: fr-FR
ms.lasthandoff: 04/23/2019
ms.locfileid: "62003689"
---
# <a name="catch-block-never-reached-because-name1-inherits-from-name2"></a>Bloc 'Catch' jamais atteint, car '\<nom1 >' hérite de '\<nom2 >'
Un bloc `Catch` du code n’est jamais atteint.  
  
 Par défaut, ce message est un avertissement. Pour plus d’informations sur le masquage des avertissements ou le traitement des avertissements en tant qu’erreurs, consultez [Configuring Warnings in Visual Basic](/visualstudio/ide/configuring-warnings-in-visual-basic).  
  
 **ID d’erreur :** BC42029  
  
## <a name="to-correct-this-error"></a>Pour corriger cette erreur  
  
- Supprimez le bloc `Catch` redondant.  
  
## <a name="see-also"></a>Voir aussi

- [Comment utiliser le bloc try/catch pour intercepter des exceptions](../../standard/exceptions/how-to-use-the-try-catch-block-to-catch-exceptions.md)
- [Try...Catch...Finally (instruction)](../../visual-basic/language-reference/statements/try-catch-finally-statement.md)
